app/schema.graphql

```diff
@@ -576,6 +576,7 @@ enum SpanKind {
   retriever
   embedding
   agent
+  reranking
```

Comment:
minor nit that we already have `retriever` - it might make more sense for this to be `reranker` to keep in the same naming family
```
reranker = LLMRerank(choice_batch_size=5, top_n=reranker_top_n, service_context=service_context)
```

Reply:
https://python.langchain.com/docs/integrations/retrievers/cohere-reranker

Reply:
it's been renamed